### PR TITLE
Add ClusterOperatorLoggingController

### PR DESCRIPTION
**Pull request by benjaminapetersen**

### CODE REVIEW

**Comment by jhadvig** [comment]
flake
/retest

File: pkg/console/starter/starter.go [hunk 1]
```diff
@@ -5,6 +5,8 @@ import (
 	"os"
 	"time"
 
+	"github.com/openshift/library-go/pkg/operator/loglevel"
```
Comment (jhadvig) [inline pkg/console/starter/starter.go]: nit:
let's put this to the other `openshift` libs

**Comment by jhadvig** [comment]
/hold
sorry for this.. found a small nit.

File: pkg/console/starter/starter.go (hunk 1 above)
Comment (benjaminapetersen) [reply pkg/console/starter/starter.go]: done!

**Pull request merged**